website/project/model.py

```diff
@@ -890,6 +882,8 @@ def institution_relationship_url(self):
 
     def __init__(self, *args, **kwargs):
 
+        kwargs.pop('logs', [])
```

Comment:
More information: 
On `__init__`, kwargs are all the fields on the node.  Popping off `logs` handles projects that were created before the migration.

If you do not pop the logs after the migration, you will get `AttributeError: Guid object has no attribute referent`.